homeassistant/components/motion_blinds/cover.py

```diff
@@ -84,6 +83,10 @@
     vol.Optional(ATTR_WIDTH): vol.All(cv.positive_int, vol.Range(max=100)),
 }
 
+UPDATE_DELAY_STOP = 3
+UPDATE_INTERVAL_MOVING = 5
+UPDATE_INTERVAL_MOVING_WIFI = 45
+
```

Comment:
Is there a reason for these to be moved out of const.py?

Reply:
From previous reviews, I got the feedback that if a constant is only used in 1 file, it schould be placed in that file instead of a central const.py file. const.py is apperently only for constants used in multiple files.

But I would be happy to move them all back to const.py, just give the word.

Reply:
Ok understood.  This is probably debatable.  My view is that if constants relate to one another (e.g. time intervals) they should be together to aid maintainability.

But not essential.

Reply:
@davet2001 I have moved the constant to const.py